lua/lang/dart.lua

```diff
@@ -2,12 +2,32 @@ local M = {}
 
 M.config = function()
   O.lang.dart = {
+    flutter_tools = {
+      active = false,
+    },
     sdk_path = "/usr/lib/dart/bin/snapshots/analysis_server.dart.snapshot",
     formatter = {
       exe = "dart",
       args = { "format" },
     },
   }
+  if O.lang.dart.flutter_tools.active then
+    O.plugin.which_key.mappings["F"] = {
```

Comment:
@UltimateOmega  change this to line and check it fixes the keybinding not working issue
```lua
O.user_which_key = {
   F = {
     name = "+Flutter",
     c = { ":FlutterCopyProfilerUrl<CR>", "Copy Profile Url" },
     -- ...
   }
}
```

Reply:
done!

Reply:
but it's now showing up on every file type.

Reply:
even when
O.lang.dart.flutter_tools.active = false

Reply:
man i hate stylua...